test/JDBC/input/BABEL-4279-vu-verify.mix

```diff
@@ -0,0 +1,21 @@
+-- psql
```

Comment:
Let's add this test cases into all the upgrade schedule files to make sure it runs in upgrade context as well.

Reply:
Resolved.